opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -139,6 +139,109 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
+class StatusCanonicalCode(enum.Enum):
+    """Represents the canonical set of status codes of a finished Span, following the Standard GRPC codes. See
+    https://github.com/open-telemetry/opentelemetry-specification/blob/2dbae9a491224f1fddfa2bb05c2a1a444c623077/specification/api-tracing.md#statuscanonicalcode.
```

Comment:
Ideally the spec repo would get a tag with each release and we could reference that instead of some random git rev.

Might as well also reference https://github.com/grpc/grpc/blob/master/doc/statuscodes.md here and save people a click.

Reply:
Agreed, but the last OTEL specification tag was 0.1 months ago. Is it time to publish a new one?

Reply:
We should really not reference the grpc status codes, as they are not identical (unknown vs UnknownError). Also they may develop differently in the future.

Reply:
I removed links to specification, several people already commented about this with different opinions is just cleaner to have our own canonical codes.